platforms-maturity-model/v1/index.md

```diff
@@ -0,0 +1,482 @@
+* The team wants to capture more data automatically, so they look for opportunities for easy collection such as test failures in CI. However not every team uses the same CI automation so the data is only available for Java applications even though some teams have moved on to writing their services in Scala.
+
+##### Level 2, Operationalized - Consistent collection
+
+Organization has an intentional goal to verify platform products meet the needs of their market of internal users. Actionable, structured collection of user feedback is valued. Dedicated teams or individuals might be assigned to gather feedback, ensuring a more consistent approach. Feedback channels, such as surveys or user forums, are standardized, and feedback is categorized and prioritized. Beyond user feedback, there is also an expectation that user experiences are instrumented to generate usage data over time.
+
+Challenges remain in translating feedback into actionable tasks. While there is a growing repository of user data, the organization might need help effectively understanding and integrating this feedback into a platform roadmap. It may be hard to ensure that users see tangible changes driven by their feedback.
+
+###### Characteristics:
+
+* Data collection is discussed as part of most major planning sessions or capability implementations
+* Users can track their feedback in relation to platform development and can see how their feedback is incorporated into the roadmap
+* May not be alignment on exactly what to measure to verify success
+* Platform features can be measured for success, such as by measuring  user adoption or user time saved
+
+###### Examples Scenarios:
+
+* A platform team allocates 20% of their time to user defined features, which they identify based on surveys and other interview techniques. Their findings are collected into a tool that enables additional voting and commenting to further refine priorities. During implementation the requesting users are approached for collaboration on early designs and implementations. Once implemented, there are announcements which make sure requesting users are aware of new features and supported in adopting them.
+* The team focused on software delivery capabilities wants to capture more data automatically including cycle time which they automate through the build tool from commit to production. There is an understanding that cycle time can include other activities like PR review, but that isn’t included at this time.
+
+##### Level 3, Scalable - Insights
+
+While robust, standard feedback mechanisms already exist, at this stage specific data is collected in specific ways to yield specific strategic insights and actions. Specific desired results and outcomes are identified and standard metrics chosen to indicate success. Industry frameworks and standards may be used to benefit from industry research on the impact of certain behaviors.
```

Comment:
Is the overuse of the word "specific" here on purpose? It reads a bit too pointedly IMHO.

Reply:
Great catch! added some reword to the whole paragraph